MimeUtility.cs

```diff
@@ -6,41 +6,31 @@
 namespace MimeMapping
 {
     /// <summary>
-    /// MIME type utility to lookup by file extension
+    ///     MIME type utility to lookup by file extension
     /// </summary>
     public static class MimeUtility
     {
         /// <summary>
-        /// The "octet-stream" subtype is used to indicate that a body contains arbitrary binary data.
-        /// See <a href="https://www.iana.org/assignments/media-types/application/octet-stream">application/octet-stream</a>
+        ///     The "octet-stream" subtype is used to indicate that a body contains arbitrary binary data.
+        ///     See <a href="https://www.iana.org/assignments/media-types/application/octet-stream">application/octet-stream</a>
         /// </summary>
         public const string UnknownMimeType = "application/octet-stream";
 
-        static Lazy<ReadOnlyDictionary<string, string>> _lazyDict = new Lazy<ReadOnlyDictionary<string, string>>(
-            () => new ReadOnlyDictionary<string, string>(KnownMimeTypes.ALL_EXTS.Value.ToDictionary(e => e, e => KnownMimeTypes.LookupType(e)))
-        );
+        private static readonly Lazy<ReadOnlyDictionary<string, string>> LazyDict = new Lazy<ReadOnlyDictionary<string, string>>(() => new ReadOnlyDictionary<string, string>(KnownMimeTypes.ALL_EXTS.Value.ToDictionary(e => e, KnownMimeTypes.LookupType)));
 
         /// <summary>
-        /// Dictionary of all available types (lazy loaded on first call)
+        ///     Dictionary of all available types (lazy loaded on first call)
         /// </summary>
-        public static ReadOnlyDictionary<string, string> TypeMap => _lazyDict.Value;
+        public static ReadOnlyDictionary<string, string> TypeMap => LazyDict.Value;
```

Comment:
Can the whitespace and stylist changes be reverted?

Reply:
Can do!